package.json

```diff
@@ -31,21 +28,19 @@
   ],
   "readmeFilename": "README.md",
   "devDependencies": {
-    "babel": "4.7.3",
-    "glob": "^4.3.5",
-    "jshint": "^2.6.0",
-    "nyc": "^3.0.1",
-    "tap": "^1.3.1"
+    "glob": "^7.1.6",
+    "jshint": "^2.11.0",
```

Comment:
are we using linting anywhere?

Reply:
It seems this dependency was not removed when linting was removed. https://github.com/krakenjs/confit/commit/39a3963f2444018fae3d0d1607afb9eb8a84ce21. Maybe for IDE?